pkg/controller/machine/controller.go

```diff
@@ -323,20 +322,24 @@ func (r *ReconcileMachine) drainNode(machine *machinev1.Machine) error {
 		return fmt.Errorf("unable to get node %q: %v", machine.Status.NodeRef.Name, err)
 	}
 
-	if err := kubedrain.Drain(
-		kubeClient,
-		[]*corev1.Node{node},
-		&kubedrain.DrainOptions{
-			Force:              true,
-			IgnoreDaemonsets:   true,
-			DeleteLocalData:    true,
-			GracePeriodSeconds: -1,
-			Logger:             info.New(klog.V(0)),
-			// If a pod is not evicted in 20 second, retry the eviction next time the
-			// machine gets reconciled again (to allow other machines to be reconciled)
-			Timeout: 20 * time.Second,
-		},
-	); err != nil {
+	drainer := &drain.Helper{
```

Comment:
We are missing some options here.

https://github.com/kubernetes-sigs/cluster-api/blob/master/controllers/machine_controller.go#L330-L348